components/file_system/src/io_stats/mod.rs

```diff
@@ -53,6 +53,7 @@ pub use proc::*;
 // See https://man7.org/linux/man-pages/man2/open.2.html#NOTES
 #[cfg(test)]
 #[repr(align(512))]
+#[cfg_attr(not(target_os = "linux"), allow(unused))]
```

Comment:
Unrelated change?

Reply:
This is to fix the clippy warning on non-linux targets.